src/utils/ctl/cart_ctl.c

```diff
@@ -701,6 +716,7 @@ ctl_init()
 	D_ASSERTF(rc == 0, "crt_finalize() failed. rc: %d\n", rc);
 
 	d_log_fini();
+	dc_mgmt_fini();
```

Comment:
this should be   if (ctl_gdata.cg_use_daos_agent_env) {  dc_mgmt_fini(); }

also should be moved before d_log_fini()

Reply:
38f8e41f157ae7b000b47dd0bd1eba0641f39a35